backpack/__init__.py

```diff
@@ -160,10 +203,16 @@ def hook_run_extensions(module, g_inp, g_out):
         memory_cleanup(module)
 
 
-def run_extension_hook(module):
+def post_extension_hook(module: Module) -> None:
     """Execute the post extensions hook on a module after all BackPACK extensions.
 
     See the `post_backward_hook` argument of the `backpack` context manager for details.
+
+    Args:
+        module: current module
+
+    Raises:
+        RuntimeError: if any error occurred during the backward hook
```

Comment:
I think there is no need anymore to specifically raise an error in the body. The traceback will indicate that an error occured in `post_extension_hook`. This should be enough. Remove the `try ... except`. We could even inline the remaining call `CTX.get_extension_hook()(module)` in `hook_run_extensions`

Reply:
I deleted the method and put it inline.